pkg/custompluginmonitor/README.md

```diff
@@ -0,0 +1,7 @@
+# Custom Plugin Monitor
```

Comment:
We should complete the document. I'm fine with doing that in a following PR.

Reply:
Agreed on detail this in another PR.

Reply:
Fired #150 to track unresolved comments. :)